Ruby-Project/teacher_class.rb

```diff
@@ -0,0 +1,8 @@
+require './person_class'
+class Teacher < person
+  def initialize
+    super(@specialization)
+    @specialization = specialization
+    @can_use_services = true
```

Comment:
- [ ] According to the requirements can_use_services? should be a method which returns true not an instance variable please change this.